File: Web/src/domain/instrument/harmonicaPosition.ts
export type HarmonicaPosition = 'first' | 'second' | 'third'

export function semitonesAboveTheHarmonica(position: HarmonicaPosition): number {
    switch (position) {
        case 'first': return 0
        case 'second': return 7
        case 'third': return 2
    }
}

File: Core/Sources/HarmonicaCore/Entities/Instrument/HarmonicaPosition.swift
public enum HarmonicaPosition: Sendable {
    case first
    case second
    case third

    public var semitonesAboveTheHarmonica: Int {
        switch self {
        case .first: 0
        case .second: 7
        case .third: 2
        }
    }
}
